10x/count/count.wdl

```diff
@@ -1117,6 +1146,7 @@ task count_genes_split {
   File chemistry_def
 
   # get required disk size
+  # minimum of 1 plus reference and 20% wiggle room plus double the input fastq file size
   Int input_file_size
   Int required_file_size = 1 + ceil(size(reference_path, "G") * 1.2 + (input_file_size * 2))
```

Comment:
@ambrosejcarr could you rename required_file_size to required_disk_size everywhere? Seems more appropriate.